pkg/http/middlewares/authorization/timestamp.go

```diff
@@ -37,21 +37,25 @@ func (t *Timestamp) UnmarshalJSON(data []byte) error {
 	var err error
 	var numeric json.Number
 
-	err = json.Unmarshal(data, &numeric)
-	if err != nil {
-		return err
-	}
-
-	timestampStr := numeric.String()
+	timestampStr := string(data)
 	switch {
 	case strings.Contains(timestampStr, "T"):
+		timestampStr = strings.Trim(timestampStr, "\"")
 		t.time, err = time.Parse(time.RFC3339, timestampStr)
 	case strings.Contains(timestampStr, "."):
 		var v float64
+		err = json.Unmarshal(data, &numeric)
+		if err != nil {
+			return err
+		}
 		v, err = numeric.Float64()
 		t.time = timeFromFloat64(v)
 	default:
 		var v int64
+		err = json.Unmarshal(data, &numeric)
+		if err != nil {
+			return err
+		}
```

Comment:
Changes in this file were not mentioned in the PR description, was it included on purpose or by accident?

Reply:
yes, when i bumped to go 1.14, this parsing broke